css/index.css

```diff
@@ -101,17 +139,121 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+/*Boxes*/
+
```

Comment:
The parent of the boxes should contain all the styling each individual box needs.  The only thing that should change in the individual boxes is overriding the color.